src/ActivitiesSDKAdapter.js

```diff
@@ -127,4 +127,46 @@ export default class ActivitiesSDKAdapter extends ActivitiesAdapter {
 
     return action$;
   }
+
+  /**
+   * Posts an activity and returns an observable to the new activity data
+   *
+   * @param {object} activity  The activity to post
+   * @returns {Observable.<Activity>} Observable that emits the posted activity (including id)
+   */
+  postActivity(activity) {
+    const cardAttachment = activity.card && [
```

Comment:
cardAttachment represents a single attachment, so the square brackets should go on line 147.

Reply:
or

```js

const attachments = activity.card ? [{
  contentType,
  content
}] : activity.attachments;